src/main/java/service/CsvTransactionService.java

```diff
@@ -0,0 +1,31 @@
+package service;
+
+import dao.TransactionsDao;
+import java.util.List;
+import model.FruitTransaction;
+
+public class CsvTransactionService implements Processor {
+    private static final String INPUT_FILE_NAME = "inputFile";
+
+    private final TransactionsDao transactionsDao;
+    private final CsvReadService csvReadService;
+    private final CsvParseService csvParserService;
```

Comment:
avoid dependencies between classes. Create instances of classes in the Main class and pass data between them. See the example in the task description.